app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="container-fluid text-center mt-5">
+  <h2>Sign up</h2>
+  
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+  
+    <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+    </div>
+  
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+  
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password" %>
+    </div>
+  
+    <div class="field">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+    </div>
+  
+    <div class="actions">
```

Comment:
- [ ] Good job so far 👏 👏 However, I would suggest that you improve your design on the registration page. I have made the comparison below 👇 

**Given design**
![image](https://user-images.githubusercontent.com/90222110/209350334-47b2ad61-9591-4109-9d59-fb42278a874a.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/209350508-5172880f-9027-4b25-b5b2-df41de4e6633.png)

Please improve it and make sure that you stick to the design given 😃 👍